hack/generated/pkg/testcommon/kube_test_context_helpers.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package testcommon
+
+import (
+	"testing"
+	"time"
+
+	"github.com/Azure/azure-service-operator/hack/generated/_apis/microsoft.resources/v1alpha1api20200601"
+	"github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+)
```

Comment:
Rather than having an "extensions" file here, would it make more sense to just split the `KubePerTestContext` definition out into its own file and then move these things there with it?

It feels sorta awkward splitting these methods off from the methods defined in the file with `KubePerTestContext`?

Reply:
Done.